account_invoice_import_invoice2data/models/invoice2data_product_mapping.py

```diff
@@ -0,0 +1,29 @@
+# coding: utf-8
+# Copyright (C) 2021 Essent <http://www.essent.be>
+# @author Stefan Rijnhart <stefan.rijnhart@dynapps.be>
+# @author Robin Conjour <r.conjour@essent.be>
+# @author Pieter Paulussen <pieter.paulussen@dynapps.be>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from openerp import api, fields, models
+
+
+class Invoice2dataProductMapping(models.Model):
+    _name = 'invoice2data.product.mapping'
+    _description = 'Productmapping for invoice2data products'
+
+    account_analytic_id = fields.Many2one(
+        'account.analytic.account', 'Analytic Account')
+    account_id = fields.Many2one(
+        'account.account', 'Account')
+    product_id = fields.Many2one('product.product')
+    rec_name = fields.Char()
```

Comment:
What does `rec_name` mean? Can you clarify the name and/or help text of the field?

Reply:
I will add an "help" parameter, rec_name is used to match product lines ( recognize name).

Reply:
Renamed to `recogintion_string` and added a help text